purchase_batch_invoicing/wizards/purchase_batch_invoicing.py

```diff
@@ -101,9 +105,15 @@ def action_batch_invoice(self):
     @api.model
     def cron_invoice_all_pending(self, grouping="partner_id"):
         """Invoice all pending purchase orders."""
+        _logger.info("Starting to invoice all pending purchase orders.")
         wizard = self.create({
             "purchase_order_ids": self.env["purchase.order"].search(
                 self._purchase_order_domain()).ids,
             "grouping": grouping,
         })
-        return wizard.action_batch_invoice()
+        try:
+            wizard.action_batch_invoice()
+        except UserError as error:
+            _logger.warn(error.name)
```

Comment:
I prefer that this is put as info only. Warnings are considered by runbot also, and GitHub CI will show error on this warning.